src/bfbs_gen_nim.cpp

```diff
@@ -77,7 +77,6 @@ Namer::Config NimDefaultConfig() {
            /*filename_extension=*/".nim" };
 }
 
-const std::string Indent = "  ";
```

Comment:
Why this change? I don't think it is causing the nim test failure but I don't think it should be in this PR.

Reply:
It's not causing the nim test failure, but it did cause a build failure with clang on linux. Happy to take it out if needed.

Reply:
Ok I've reverted the removal of that line

Reply:
Thanks for jumping in on this btw @bjornharrtell 👍